tox.ini

```diff
@@ -1,3 +1,4 @@
 [flake8]
 max-line-length = 100
 max-doc-lentgh = 100
+ignore = E123, E125, E251, W503, E731, E741
```

Comment:
Can you add a detailed comment about why each of these are ignored?  Can be a simple as "yapf formatting causing this check to be violated", etc.

Reply:
Done